tests/openvino/test_modeling.py

```diff
@@ -1934,6 +1934,39 @@ def test_compare_to_transformers(self, model_arch):
             f"generation config : {gen_config}, transformers output {transformers_outputs}, ov_model output {ov_outputs}",
         )
 
+        # previous run was with legacy processing, one more run with features concatenation on preprocessing level
+        if (
+            model_arch in ["llava", "llava-next"]
+            and is_transformers_version(">=", "4.45")
+            and (processor.patch_size is None or processor.vision_feature_select_strategy is None)
+        ):
+            processor.patch_size = ov_model.config.vision_config.patch_size
+            processor.vision_feature_select_strategy = ov_model.config.vision_feature_select_strategy
+            if model_arch == "llava":
+                # testing model for llava does ot have specified image_seq_length and it is different from default
+                transformers_model.config.image_seq_length = 225
+                ov_model.config.image_seq_length = 225
+            self.assertTrue(processor.patch_size is not None)
+            self.assertTrue(processor.vision_feature_select_strategy is not None)
+            inputs = processor(images=self.IMAGE, text=prompt, return_tensors="pt")
+            self.assertTrue(
+                (inputs.input_ids == ov_model.config.image_token_index).sum(1).max()
+                >= ov_model.config.image_seq_length
+            )
+            set_seed(SEED)
+            with torch.no_grad():
+                transformers_outputs = transformers_model(**inputs)
+            set_seed(SEED)
+            ov_outputs = ov_model(**inputs)
+            self.assertTrue(torch.allclose(ov_outputs.logits, transformers_outputs.logits, atol=1e-4))
+            set_seed(SEED)
+            ov_outputs = ov_model.generate(**inputs, generation_config=gen_config)
+            set_seed(SEED)
+            transformers_outputs = transformers_model.generate(**inputs, generation_config=gen_config)
```

Comment:
Shouldn't this be called inside `with torch.no_grad():` as above?

Reply:
yes, fixed